public/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -195,12 +195,19 @@ There are multiple consensus clients (previously known as 'Eth2' clients) to sup
 
 | Client                                                        | Language   | Operating systems     | Networks                                                          |
 | ------------------------------------------------------------- | ---------- | --------------------- | ----------------------------------------------------------------- |
+| [Grandine](https://docs.grandine.io/)                         | Rust       | Linux, Windows, macOS | Beacon Chain, Goerli, Sepolia, and more                           |
```

Comment:
Please move this to last and mark with `(beta)` at the end given we are not yet at a v1 production release.